mne/viz/_3d.py

```diff
@@ -1029,38 +1159,77 @@ def plot_source_estimates(stc, subject=None, surface='inflated', hemi='lh',
                 construction to obtain negative control points.
 
     cortex : str or tuple
-        specifies how binarized curvature values are rendered.
-        either the name of a preset PySurfer cortex colorscheme (one of
-        'classic', 'bone', 'low_contrast', or 'high_contrast'), or the
-        name of mayavi colormap, or a tuple with values (colormap, min,
-        max, reverse) to fully specify the curvature colors.
+        Specifies how binarized curvature values are rendered.
+        Either the name of a preset PySurfer cortex colorscheme (one of
+        'classic', 'bone', 'low_contrast', or 'high_contrast'), or the name of
+        mayavi colormap, or a tuple with values (colormap, min, max, reverse)
+        to fully specify the curvature colors. Has no effect with mpl backend.
     size : float or pair of floats
         The size of the window, in pixels. can be one number to specify
         a square window, or the (width, height) of a rectangular window.
+        Has no effect with mpl backend.
     background : matplotlib color
         Color of the background of the display window.
     foreground : matplotlib color
-        Color of the foreground of the display window.
+        Color of the foreground of the display window. Has no effect with mpl
+        backend.
     initial_time : float | None
         The time to display on the plot initially. ``None`` to display the
         first time sample (default).
     time_unit : 's' | 'ms'
         Whether time is represented in seconds ("s", default) or
         milliseconds ("ms").
+    backend : 'auto' | 'mayavi' | 'matplotlib'
+        Which backend to use. If ``'auto'`` (default), tries to plot with
+        mayavi, but resorts to matplotlib if mayavi is not available.
+
+        .. versionadded:: 0.15.0
 
+    spacing : str
+        The spacing to use for the source space. Can be ``'ico#'`` for a
+        recursively subdivided icosahedron, ``'oct#'`` for a recursively
+        subdivided octahedron, or ``'all'`` for all points. In general, you can
+        speed up the plotting by selecting a sparser source space. Has no
+        effect with mayavi backend. Defaults  to 'oct5'.
+
+        .. versionadded:: 0.15.0
 
     Returns
     -------
     brain : Brain
-        A instance of surfer.viz.Brain from PySurfer.
-    """
-    import surfer
-    from surfer import Brain, TimeViewer
-    import mayavi
-
+        A instance of surfer.viz.Brain from PySurfer or matplotlib figure.
```

Comment:
these types should be in the `: ` above, not (just) here (`Brain` above is now a misnomer)